app/src/main/java/me/ccrama/redditslide/SpoilerRobotoTextView.java

```diff
@@ -83,7 +59,9 @@
     private              List<Integer>        storedSpoilerStarts = new ArrayList<>();
     private              List<Integer>        storedSpoilerEnds   = new ArrayList<>();
     private static final Pattern              htmlSpoilerPattern  =
-            Pattern.compile("<a href=\"([#/](?:spoiler|sp|s))\">([^<]*)</a>");
+            Pattern.compile("<a.*href=\"[#/](?:spoiler|sp|s)\".*?>(.*?)</a>");
+    private static final Pattern nativeSpoilerPattern =
+            Pattern.compile("<span.*class=.*?md-spoiler-text.*?>(.*?)</span>");
```

Comment:
Same problem with the `.*` in this pattern, the `.*?` between `class` and `md-spoiler-text` may also over-track if reddit has or introduces (I don't think it does yet?) any other formatting using spans

Reply:
@Alexendoo I think you are correct about spans, but they could definitely add something without notice and cause problems